### PR TITLE
`convertEasyEdaJsonToCircuitJson`: convert oval pads with hole radius to pill-shaped plated holes

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
Closes https://github.com/tscircuit/easyeda-converter/issues/81

### CODE REVIEW

File: tests/convert-to-ts/__snapshots__/C165948-to-ts.snap.svg
Comment (andrii-balitskyi) [inline tests/convert-to-ts/__snapshots__/C165948-to-ts.snap.svg:1]: Stroke width here should be fixed by https://github.com/tscircuit/easyeda-converter/pull/84

**Review by seveibar — APPROVED** [review]
Nice. It seems like we may have an issue in circuit to svg because the hole isnt tall enough? It should look almost like a coin slot

**Comment by andrii-balitskyi** [comment]
@seveibar not sure about that. Could an issue in circuit to svg. But to be sure, am I calculating the props correctly [here](https://github.com/tscircuit/easyeda-converter/pull/83/files#diff-65e54d21100453a1a44586f63df2908b89b733f7865f2cbd26d66aeabeae0288R166-R169)?

File: tests/convert-to-ts/__snapshots__/C165948-to-ts.snap.svg
Comment (andrii-balitskyi) [reply tests/convert-to-ts/__snapshots__/C165948-to-ts.snap.svg:1]: Updated in 7b4065040015e69a32d842c800654e2bc25e72f9

**Pull request merged**